src/app/ApplicationTemplate.UWP/Views/Styles/Controls/PathControl.xaml

```diff
@@ -1,8 +1,11 @@
-﻿<ResourceDictionary
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" 
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-	xmlns:u="using:Nventive.View.Controls">
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:u="using:Nventive.View.Controls">
 	
+	<!-- KNOWN ISSUE : It is impossible to add a color resource in a setter. -->
```

Comment:
It is actually possible because it works with TextBlock. It's also not specific to colors and brushes.